src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -169,8 +175,12 @@ private void handleExit() {
         primaryStage.hide();
     }
 
-    public PersonListPanel getPersonListPanel() {
-        return personListPanel;
+    public VolunteerListPanel getVolunteerListPanel() {
+        return volunteerListPanel;
+    }
+
+    public ElderlyListPanel getElderlyListPanel() {
+        return elderlyListPanel;
     }
 
```

Comment:
Just curious, because for the screenshot attached to the commit, I am not sure how are the 3 separate lists distingushed; would it be possible to color the 3 boxes, so that we can distingush what are the boundaries of the 3 list panels?

Reply:
Can be done in future iterations